api/SubUpdaters/PublicBidSubUpdater.js

```diff
@@ -0,0 +1,72 @@
+const { DateTime } = require('luxon');
+const MessageSender = require('discord-helper-lib/MessageSender.js');
+const MessageColors = require('discord-helper-lib/MessageColors');
+const MessageWithEmbed = require('discord-helper-lib/MessageWithEmbed.js');
+const DateCleaner = require('../DateCleaner.js');
+const ScoreRounder = require('../ScoreRounder.js');
+
+class PublicBidSubUpdater {
+    constructor() {
+        this.MessageSender = new MessageSender();
+        this.MessageColors = new MessageColors();
+    }
+
+    async publicBidUpdate(leagueYear, lastCheckDate, channelToSend) {
+        if (!leagueYear.publicBiddingGames || leagueYear.publicBiddingGames.masterGames.length === 0) {
+            return;
+        }
+
+        if (lastCheckDate > DateTime.fromISO(leagueYear.publicBiddingGames.postedTimestamp)) {
+            return;
+        }
+
+        let gameMessages = [];
+        for (const publicBid of leagueYear.publicBiddingGames.masterGames) {
+            let gameMessage = '';
+            const releaseDate = DateCleaner.clean(publicBid.masterGame.estimatedReleaseDate);
+            gameMessage += `**${publicBid.masterGame.gameName}**`;
+
+            if (publicBid.counterPick) {
+                gameMessage += ' (🎯 Counter Pick Bid)';
+            }
+
+            gameMessage += '\n';
+            gameMessage += `> Release Date: ${releaseDate}\n`;
+
+            const roundedHypeFactor = ScoreRounder.round(publicBid.masterGame.dateAdjustedHypeFactor, 1);
+            gameMessage += `> Hype Factor: ${roundedHypeFactor}\n`;
+            gameMessages.push(gameMessage);
+        }
+
+        const leagueLink = `https://www.fantasycritic.games/league/${leagueYear.leagueId}/${leagueYear.leagueYear}`;
+        const finalMessage = gameMessages.join('\n') + `\n[Visit League Page](${leagueLink})`;
+        const lastSunday = this.getLastSunday(new Date());
+        const dateString = lastSunday.toLocaleDateString('en-us', {
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+        });
+        const header = `Public bids (Week of ${dateString})`;
+
+        const messageToSend = new MessageWithEmbed(
+            finalMessage,
+            header,
+            null,
+            null,
+            null,
+            this.MessageColors.RegularColor,
+            null,
+            null
```

Comment:
```suggestion
            leagueLink
```

Reply:
Would it make more sense here to put this link in the title as well, similar to GetLeague?